presto-main/src/main/java/com/facebook/presto/operator/TableFinishOperator.java

```diff
@@ -42,21 +45,30 @@
         private final int operatorId;
         private final PlanNodeId planNodeId;
         private final TableFinisher tableFinisher;
+        private final OperatorFactory statisticsAggregation;
```

Comment:
s/statisticsAggregation/operatorFactory?

Reply:
I think `statisticsAggregation` better describes what it is in the context of the `TableFinishOperator`